config-serializer/config-items/Layer.ts

```diff
@@ -5,6 +5,7 @@ import {UhkBuffer} from '../UhkBuffer';
 export class Layer extends Serializable<Layer> {
 
     modules: Modules;
+    animation: string = 'none';
```

Comment:
We should restrict the animation type with the valid ones. Typescript allows the define a type as a set of strings like:
`animation: 'none' | 'leftIn' | 'leftOut' | 'rightIn' | 'rightOut';`. The correct types should be defined elsewhere not inlined as a described here, but right now I don't remember the correct syntax to define it with `type`.